src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -55,6 +55,7 @@ const StyledTableToolbarContent = styled(TableToolbarContent)`
     padding-right: 1px;
     align-items: center;
     font-size: 0.875rem;
+    height: 2.25rem;
```

Comment:
I think that the right value here would be `3.125rem` to be equivalent to `50px`.